### PR TITLE
Added runtime_config to env var mapping

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: builder/gen-dockerfile/tests/test_data/values_on_both/app.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  front_controller_file: app.php
+  nginx_conf_http_include: files/nginx-http.conf
+  nginx_conf_include: files/nginx-app.conf
+  nginx_conf_override: files/nginx.conf
+  php_fpm_conf_override: files/php-fpm.conf
+  php_ini_override: files/php.ini
+  supervisord_conf_addition: files/additional-supervisord.conf
+  supervisord_conf_override: files/supervisord.conf
+
+env_variables:
```
Comment (chingor13) [inline builder/gen-dockerfile/tests/test_data/values_on_both/app.yaml]: At runtime, these will be set by App Engine and overwrite the ones set in the runtime config. Is this something we want to warn or even fail the build for?

**Comment by chingor13** [comment]
Also, maybe update the README.md?

File: builder/gen-dockerfile/tests/test_data/values_on_both/app.yaml (hunk 1 above)
Comment (tmatsuo) [inline builder/gen-dockerfile/tests/test_data/values_on_both/app.yaml]: Yeah let's fail when both have values

Comment (tmatsuo) [inline builder/gen-dockerfile/tests/test_data/values_on_both/app.yaml]: Done. Now it fails when there are values on both.
@chingor13 PTAL

**Pull request merged**